src/amo/components/VPNPromoBanner/styles.scss

```diff
@@ -0,0 +1,141 @@
+@import '~amo/css/styles';
+
+$cta-text-color: #1d1133;
+$cta-alternate-color: #f9f7fd;
+
+.VPNPromoBanner {
+  background-color: #c5c8fb;
+  font-family: sans-serif;
+  font-size: 16px;
+  line-height: $line-height-compressed;
+  max-height: 500px;
+  overflow: hidden;
+  position: relative;
+  transition: max-height 0.3s ease-out;
```

Comment:
I don't understand what this transition does.

Reply:
I don't either. I guess we can remove it, in which case we can also remove the `@media (prefers-reduced-motion)` below, and then not worry about it not being applied across all transitions.